Revit_oM/Interface/IInstance.cs

```diff
@@ -23,17 +23,21 @@
 using BH.oM.Adapters.Revit.Properties;
 using BH.oM.Base;
 using BH.oM.Geometry;
+using System.ComponentModel;
 
 namespace BH.oM.Adapters.Revit.Interface
 {
+    [Description("A generic wrapper BHoM interface corresponding to any Revit element.")]
```

Comment:
Would maybe change this to say that this is the base interface for the wrapper classes that can wrap any revit object, rather than stating that this interface can do the job by itself (as it kind of reads atm).